packages/components/src/components/domainproperties/issues/IssuesPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,221 @@
+import React from 'react';
+import { Col, FormControl, Row } from 'react-bootstrap';
+
+import { List } from 'immutable';
+
+import { SectionHeading } from '../SectionHeading';
+
+import { DomainFieldLabel } from '../DomainFieldLabel';
+import { Principal, SelectInput } from '../../..';
+
+import { IssuesModel } from './models';
+import {UserGroup} from "../../permissions/models";
+
+interface IssuesBasicPropertiesInputsProps {
+    model: IssuesModel;
+    onInputChange: (any) => void;
+}
+
+interface SecurityUserGroupProps {
+    model: IssuesModel;
+    coreGroups?: List<Principal>;
+    onSelect?: (selected: Principal) => any;
+    coreUsers?: List<UserGroup>;
+}
+
+export class IssuesBasicPropertiesFields extends React.PureComponent<IssuesBasicPropertiesInputsProps> {
+    render() {
+        const { model, onInputChange } = this.props;
+        return (
+            <Col xs={11} md={6}>
+                <SectionHeading title="Basic Properties" />
+                <CommentSortDirectionDropDown model={model} onInputChange={onInputChange} />
+                <SingularItemNameInput model={model} onInputChange={onInputChange} />
+                <PluralItemNameInput model={model} onInputChange={onInputChange} />
+            </Col>
+        );
+    }
+}
+
+export class AssignmentOptions extends React.PureComponent<SecurityUserGroupProps> {
+    render() {
+        const { model, coreUsers, coreGroups, onSelect } = this.props;
+        return (
+            <Col xs={11} md={6}>
+                <SectionHeading title="Assignment Options" />
+                <AssignedToGroupInput
+                    model={model}
+                    coreGroups={coreGroups}
+                    onSelect={(selected: Principal) => onSelect(selected)}
+                />
+                <DefaultUserAssignmentInput model={model} coreUsers={coreUsers} onSelect={(selected: Principal) => onSelect(selected)} />
+            </Col>
+        );
+    }
+}
+
+export class SingularItemNameInput extends React.PureComponent<IssuesBasicPropertiesInputsProps> {
+    render() {
+        const { model, onInputChange } = this.props;
+        const value = model.name === null ? '' : model.name;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Singular item name" required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        id="singularItemName"
+                        type="text"
+                        placeholder="Enter a singular name for this Issue"
```

Comment:
I think we should remove this placeholder text and instead add in a help tooltip text to the DomainFieldLabel. I can help come up with the text for that tooltip.

Reply:
help tooltips are now added to all top level properties. For Singular and Plural, they are the same since they kind of go together, imo - let me know if you think they need to be separated out.

Reply:
Great. Thanks for adding those. I will review the text during manual testing.